app/controllers/admin/manager_invitations_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Admin
+  class ManagerInvitationsController < Spree::Admin::BaseController
+    def create
+      @email = params[:email]
+      @enterprise = Enterprise.find(params[:enterprise_id])
+
+      existing_user = Spree::User.where("email = :email OR unconfirmed_email = :email", email: @email).first
```

Comment:
:warning: watch out! there's no index for the `unconfirmed_email` column. We must add one.